src/main/java/net/imglib2/RealRandomAccessible.java

```diff
@@ -48,6 +48,10 @@
  * expect that the domain is infinite. (In contrast to this,
  * {@link RealRandomAccessibleRealInterval}s have a finite domain.)
  * </p>
+ * <p>
```

Comment:
I do not think this is true.  Typically, a RealRandomAccessible is expected to be defined everywhere.  Where is this not true (while not being an errror)?